wolfssl/wolfcrypt/port/atmel/atmel.h

```diff
@@ -108,21 +122,58 @@ int  atmel_get_rev_info(word32* revision);
 void atmel_show_rev_info(void);
 
 WOLFSSL_API int wolfCrypt_ATECC_SetConfig(ATCAIfaceCfg* cfg);
-
+#if defined(WOLFSSL_MICROCHIP_TA100)
+WOLFSSL_API int wc_Microchip_SetSharedDataConfig(ta_element_attributes_t* cfg);
```

Comment:
The function right about this uses wolfCrypt_ATECC_SetConfig. Lets match the naming in the same header file. wolfCrypt_*